Website/src/rendering/src/stories/Checkout/CartDetails.stories.tsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import CartDetails from '../../components/Checkout/CartDetails';
+import { MockSlice, MockStore } from '../mock-store';
+import { Spec } from 'ordercloud-javascript-sdk';
+
+export default {
+  title: 'Components/Checkout/CartDetails',
+  component: CartDetails,
+} as ComponentMeta<typeof CartDetails>;
+
+const Template: ComponentStory<typeof CartDetails> = (args) => <CartDetails {...args} />;
+
+const cartState = {
```

Comment:
@NellyKostadinova the mock consts in this file seem to be almost identical to the ones found in different stories. It would be good to extract them in a separate file to avoid repetition between stories. You can extract any other mocked const that you identify as duplicated too. Thanks.
I did something similar with `PageStoriesCommon.ts`.